cruise.umple/test/cruise/umple/compiler/487_parallelStateMachines_sameRegionSameNames_2.ump

```diff
@@ -0,0 +1,42 @@
+class X {
```

Comment:
With the current version of Umple, the following enums are generated when this file is compiled into Java:

```
// line 1 "487_parallelStateMachines_sameRegionSameNames_2.ump"
public class X
{

  //------------------------
  // MEMBER VARIABLES
  //------------------------

  //X State Machines
  public enum Sm { s0, s1 }
  public enum SmT1 { Null, t1 }
  public enum SmT2 { Null, t2, t3 }
  public enum SmS1 { Null, s2, s3 }
  public enum SmS1T1 { Null, t1, t4 }
  public enum SmS1T6 { Null, t6 }
  public enum SmS1T2 { Null, t2, t5 }
  public enum SmS1T7 { Null, t7, t8 }
  public enum SmS1T6 { Null, t6, t9 }
  private Sm sm;
  private SmT1 smT1;
  private SmT2 smT2;
  private SmS1 smS1;
  private SmS1T1 smS1T1;
  private SmS1T6 smS1T6;
  private SmS1T2 smS1T2;
  private SmS1T7 smS1T7;
  private SmS1T6 smS1T6;
```